packages/cli/src/cmds/preview/depsChecker/cliLogger.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as os from "os";
+import cliLogger from "../../../commonlib/log";
+import { LogLevel } from "@microsoft/teamsfx-api";
+import { IDepsLogger } from "./checker";
+
+export class VSCodeLogger implements IDepsLogger {
```

Comment:
Should this name be modified?

Reply:
Good catch. Renamed all "vscode" to "cli"